.github/CODEOWNERS

```diff
@@ -1,8 +1,14 @@
 #ECCN: Open Source
-#GUSINFO: Program Management Orion, SFDO Grants Management
-# 
+#GUSINFO: NPC Orion, SFDO Grants Management
+
+#
+
 # Copyright (c) 2021, salesforce.com, inc.
+
```

Comment:
@sella24 , do we really want all these spaces?

Reply:
Those are just coming from the autoformatting on this repo. I'll remove them.